src/main/scala/gpp/ui/forms/EnumViewMultipleSelect.scala

```diff
@@ -106,114 +103,24 @@ final case class EnumViewMultipleSelect[F[_], A](
   val enum:             Enumerated[A],
   val display:          Display[A],
   val effect:           Effect[F]
-) extends ReactProps[EnumViewMultipleSelect[Any, Any]](EnumViewMultipleSelect.component) {
-  def withMods(mods: TagMod*): EnumViewMultipleSelect[F, A] = copy(modifiers = modifiers ++ mods)
-}
+) extends ReactProps[EnumViewSelectBase](EnumViewSelectBase.component)
+    with EnumViewSelectBase {
 
-object EnumViewMultipleSelect {
-  type Props[F[_], A] = EnumViewMultipleSelect[F, A]
+  type AA    = A
+  type BB    = Set[A]
+  type FF[X] = F[X]
```

Comment:
Did you have troubles with dependent types?

Reply:
I just had troubles with the types, period. 😆 What were you thinking of?

Reply:
Often with dependent types you need to use the `Aux` pattern
https://www.signifytechnology.com/blog/2019/02/scala-the-aux-pattern-and-path-dependent-types-by-basement-crowd